js/mocha-tests/.editorconfig

```diff
@@ -0,0 +1,10 @@
+# http://editorconfig.org
```

Comment:
ok, I have the habit to include since I run on Windows and this makes it easier to get consistent line endings/tabs. Maybe I could just leave it with the charset and end_of_line settings?